test/git/test_invocations.py

```diff
@@ -1,14 +1,22 @@
 from __future__ import print_function
 
 import os, sys, shutil, unittest, tempfile, tarfile, virtualenv, warnings
+from wheel.pep425tags import get_abbr_impl, get_impl_ver, get_abi_tag, \
+    get_platform
 
 sys.path.insert(0, "src")
 from from_file import versions_from_file
 import common
 
+
 pyver_major = "py%d" % sys.version_info[0]
 pyver = "py%d.%d" % sys.version_info[:2]
 
+# For binary wheels with native code
+impl, impl_ver = get_abbr_impl(), get_impl_ver()
+abi, plat = get_abi_tag(), get_platform()
```

Comment:
`get_platform()` now takes an `archive_root` argument, but it can be `None`.

```suggestion
abi = get_abi_tag()
try:
    plat = get_platform(None)
except TypeError:  # wheel < 0.34
    plat = get_platform()
```